### DISCUSSION TITLE
如何在标题和文章之间新增固定广告位

### DISCUSSION THREAD

**Discussion by Zephery** [vuepress-theme-hope/vuepress-theme-hope#3876] [Q&A] [answered]
想在markdown的标题下面固定展示谷歌广告，看到有类似的文章[vuepress配置谷歌广告](https://www.sofineday.com/vuepress-vue-google-adsense.html#_1-%E7%94%B3%E8%AF%B7%E5%B9%BF%E5%91%8A%E4%BB%A3%E7%A0%81)，但是这个需要再每一篇markdown下面都添加<ArticleTopAd></ArticleTopAd>这种，有没有办法统一在配置文件里就实现。

**Comment by Mister-Hope** [comment, marked as answer]
Use slots. See:

- https://theme-hope.vuejs.press/zh/demo/slot.html
- https://theme-hope.vuejs.press/zh/guide/advanced/replace.html#%E5%A6%82%E4%BD%95%E9%80%9A%E8%BF%87%E5%88%AB%E5%90%8D%E6%9B%BF%E6%8D%A2%E7%BB%84%E4%BB%B6

### We are always expecting users to read docs carefully.

**Reply by Zephery** [reply]
感谢，已完成
https://github.com/Zephery/MyWebsite/commit/ef4428c3239eeec37bcb7aebcb93a99afabd0755
